### PR TITLE
Project-happy-thoughts, by Ulrika Öhman

### PR DESCRIPTION
**Pull request by uohman**
Link to my deployed project: 
https://loquacious-kitten-5cad9c.netlify.app/

My reflections on how this week's project turned out: 
I struggled a lot this week trying to understand changes in state and the useEffect hook. This week's example project and my teammates was very helpful though.

### CODE REVIEW

File: code/src/components/ThoughtsForm.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <div className="form-container">
+      <form onSubmit={onFormSubmit}>
+        <h1>What is making you happy right now?</h1>
+        <textarea
+          value={newThought}
+          onChange={onNewThoughtChange}
+          placeholder="Type your happy thoughts here..."
+          maxLength="140" />
+        <div className="counter">{0 + newThought.length} / {140 - newThought.length} </div>
```
Comment (KajaWilbik) [inline code/src/components/ThoughtsForm.js:13]: Nice way of making a counter. Is it ok if I borrow this idea?

File: code/src/components/ThoughtsForm.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <button
+          className="submit-btn"
+          type="submit">
+          <span className="emoji" role="img" aria-label="heart-emoji">❤️</span> Send happy thought! <span role="img" aria-label="heart-emoji">❤️</span>
```
Comment (KajaWilbik) [inline code/src/components/ThoughtsForm.js:17]: I do not understand why you put emoji in <span>?

File: code/src/normalize.css [hunk 3]
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (KajaWilbik) [inline code/src/normalize.css:1]: Are you using normalize.css to make the project adaptable to a different web browser? Am I understanding it properly?

**Review by KajaWilbik — COMMENTED** [review]
Good job! Your code is clear and easy to read, I had some questions about some parts of your code. It is amazing how your coding skills developed from our first project together. Congratulation!

File: code/public/index.html [hunk 4]
```diff
@@ -4,6 +4,10 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
[the remark this answers is not in the document]
Comment (uohman) [reply code/public/index.html:8]: Yep, they are from Google fonts. :)

File: code/src/components/ThoughtsForm.js (hunk 1 above)
Comment (uohman) [reply code/src/components/ThoughtsForm.js:13]: Of course! I borrowed it too from somewhere. :)

File: code/src/components/ThoughtsForm.js (hunk 2 above)
Comment (uohman) [reply code/src/components/ThoughtsForm.js:17]: The emoji here is only as a nice detail on the button.

File: code/src/normalize.css (hunk 3 above)
Comment (uohman) [reply code/src/normalize.css:1]: Yes, Normalize.css provides cross-browser consistency, similar to the way Reset CSS works.

**Pull request closed without merging**